views/publication/search_hit.gohtml

```diff
@@ -1,111 +1,143 @@
-                        <i class="if if-external-link"></i>
-                        <span>WoS</span>
-                    </a>
-                    {{end}}
-                    {{with .Publication.PubMedID}}
-                    <a class="dropdown-item" href="{{resolveIdentifier "PubMed" .}}" target="_blank">
-                        <i class="if if-external-link"></i>
-                        <span>PubMed</span>
-                    </a>
-                    {{end}}
-                {{else}}
-                    {{if .Publication.DOI}}
-                    <a class="dropdown-item" href="https://www.altmetric.com/explorer/outputs?scope=all&identifier={{.Publication.DOI}}" target="_blank">
-                        <i class="if if-external-link"></i>
-                        <span>Altmetric</span>
-                    </a>
-                    {{end}}
+                {{with .Publication.WOSID}}
+                <a class="dropdown-item" href="{{resolveIdentifier "WebOfScience" .}}" target="_blank">
+                    <i class="if if-external-link"></i>
+                    <span>WoS</span>
+                </a>
                 {{end}}
-                {{/*
-                {{if and (eq .Publication.Status "public") .User.ORCID .User.ORCIDToken}}
-                <button class="dropdown-item"
-                    hx-post="{{pathFor "publication_orcid_add" "id" .Publication.ID}}"
-                    hx-target="#publication-{{.Publication.ID}}-orcid-status"
-                    hx-trigger="click"
-                    hx-swap="outerHTML"
-                >
-                    <i class="if if-orcid" style="color: #A6CE39"></i>
-                    <span>Send to ORCID</span>
-                </button>
+                {{with .Publication.PubMedID}}
+                <a class="dropdown-item" href="{{resolveIdentifier "PubMed" .}}" target="_blank">
+                    <i class="if if-external-link"></i>
+                    <span>PubMed</span>
+                </a>
                 {{end}}
-                */}}
-                {{if .User.CanDeletePublication .Publication}}
-                <div class="dropdown-divider"></div>
-                <button class="dropdown-item"
-                    hx-get="{{pathFor "publication_confirm_delete" "id" .Publication.ID|querySet "redirect-url" .CurrentURL.String}}"
-                    hx-target="#modals"
-                >
-                    <i class="if if-delete"></i>
-                    <span>Delete</span>
-                </button>
+            {{else}}
+                {{if .Publication.DOI}}
+                <a class="dropdown-item" href="https://www.altmetric.com/explorer/outputs?scope=all&identifier={{.Publication.DOI}}" target="_blank">
+                    <i class="if if-external-link"></i>
+                    <span>Altmetric</span>
+                </a>
                 {{end}}
-            </div>
+            {{end}}
+            {{/*
```

Comment:
Best to leave out code in comment. Probably copied and pasted from another source? This block runs from line 52 till line 63

Reply:
never mind, was already in the block on the left, but looked a little different though

Reply:
Can I completely drop this? Or was there something else that doesn't look right? Thanks for the review.